packages/dds/tree/src/test/domains/json/jsonCursor.spec.ts

```diff
@@ -146,18 +146,18 @@ describe("JsonCursor", () => {
 
         function expectFound(cursor: ITreeCursor, key: FieldKey, index = 0) {
             assert(0 <= index && index < cursor.length(key),
-                `.length() must include index of existing child '${key}[${index}]'.`);
+                `.length() must include index of existing child '${String(key)}[${index}]'.`);
```

Comment:
Why is this conversion necessary?

Reply:
the type system indicates that key can be a symbol now, which produces an error here (it would error at runtime apparently). Since the value is always a string in this test, there no real behavioral change, its just making the type conversion ore explicit/tolerant to make the build happy.